app/profile/page.js

```diff
@@ -264,6 +269,22 @@ export default function EditProfile() {
                         </select>
                     </div>
 
+                    <div>
+                        <label htmlFor="pronouns" className="text-sm font-medium text-gray-700 block mb-2">Pronouns</label>
+                        <select
+                            id="pronouns"
+                            value={pronouns}
+                            onChange={(e) => setPronouns(e.target.value)}
+                            className="w-full p-3 border border-gray-300 rounded-md text-black"
+                        >
+                            <option value="">Select your pronouns</option>
```

Comment:
It's the child's attributes, not the user's:
```suggestion
                            <option value="">Child's gender</option>
```